src/components/TextField/TextField.tsx

```diff
@@ -189,11 +189,13 @@ export const TextField = ({
     }
   }, [ariaDescribedBy, fieldNote, id]);
 
-  const componentClassName = clsx(styles['text-field'], className, {
-    [styles['text-field--inverted']]: inverted === true,
-    [styles['eds-is-error']]: isError,
-    [styles['eds-is-disabled']]: disabled,
-  });
+  const componentClassName = clsx(
+    styles['text-field'],
+    className,
+    inverted && styles['text-field--inverted'],
+    isError && styles['eds-is-error'],
+    disabled && styles['eds-is-disabled'],
```

Comment:
I'm just now realizing it's kind of weird that we use `disabled` and not `isDisabled` 🤔

Reply:
@dierat Yeah I've thought about that conundrum before too! It is weird that it's not more explicit. Ultimately, I default to `disabled` simply because that's what the native HTML attribute is. I don't know if that's good or bad but that's what I think of anyways!